<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>704</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1038</width>
    <height>704</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1038</width>
    <height>704</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sia BATTLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/img/main_icon.png</normaloff>:/images/img/main_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="butt_connect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>873</x>
      <y>25</y>
      <width>127</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>255</width>
      <height>90</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="info_ship1_my">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>550</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/ships/img/ship1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="info_ship2_my">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>585</y>
      <width>70</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/ships/img/ship2_h.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num_ship1_my">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>555</y>
      <width>20</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Daneehand Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x4</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_ship2_my">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>590</y>
      <width>20</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Daneehand Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x3</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_ship2_my_2">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>580</y>
      <width>140</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>140</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/ships/img/ship4_h.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="num_ship4_my">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>588</y>
      <width>20</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Daneehand Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x1</string>
    </property>
   </widget>
   <widget class="QLabel" name="num_ship3_my">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>553</y>
      <width>20</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Daneehand Regular</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_ship1_my_2">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>545</y>
      <width>105</width>
      <height>35</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>105</width>
      <height>35</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/ships/img/ship3_h.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>161</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Safron Modern</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ваши корабли</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_name">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>34</y>
      <width>119</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>34</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>pass</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_Add_ships">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>26</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Astakhov Dished DL</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>1. Расставьте корабли</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_Add_name">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>62</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Astakhov Dished DL</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>2. Введите своё имя и пароль</string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_connect">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>99</y>
      <width>661</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Astakhov Dished DL</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>3. Нажмите Connect (*имя и пароль должны содержать минимум 4 символа)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="action_journal"/>
    <addaction name="actionRestart"/>
    <addaction name="action_IP"/>
   </widget>
   <addaction name="menuMENU"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Astakhov Dished DL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(185, 31, 25);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <action name="action_journal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/img/journal_ico.png</normaloff>:/images/img/journal_ico.png</iconset>
   </property>
   <property name="text">
    <string>Журнал</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="action_IP">
   <property name="text">
    <string>Обновить IP сервера</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
